x-pack/plugins/cases/server/connectors/cases/utils.ts

```diff
@@ -21,3 +21,14 @@ export const partitionRecordsByError = (
 
   return [validRecords, errors];
 };
+
+export const partitionByNonFoundErrors = <T extends Array<{ statusCode: number }>>(
```

Comment:
Was this also moved?

Reply:
No this is a new function that separates 404 errors from the rest of the other errors.